cmd/antictl/main.go

```diff
@@ -126,6 +126,31 @@ func main() {
 					},
 				},
 				// TODO (mierdin): Add command to make a session persistent
```

Comment:
Don't forget to remove this :)

Reply:
;) sure, will remove it in final version